src/app/_components/Map/Map.js

```diff
@@ -0,0 +1,17 @@
+import dynamic from 'next/dynamic';
```

Comment:
🟠 consider renaming `Map.js` => `component.js`, `MapWindow.js` => `window.js`

Reply:
Thanks for comment! This naming seems better for me. Renamed components.